specification_documents/examples/intro_qc2.mzQC

```diff
@@ -33,20 +33,30 @@
                   "value": "LTQ Orbitrap Velos"
                 }
               ]
+            },
+            {
+              "location": "file://tmp/QC2_18052020.mzid",
+              "name": "QC2_18052020",
+              "fileFormat": {
+                "accession": "MS:1002073",
+                "name": "mzIdentML format"
```

Comment:
**issue:** The mzIdentML currently doesn't seem to be recognized by the validator as a relevant ID file for ID-based metrics:

> ID based metric but no ID input file of severity 6 and message: ID based metrics present but no ID input file could be found registered in the mzQC file: run/set label =